package/MDAnalysis/core/topology.py

```diff
@@ -315,7 +315,7 @@ class Topology(object):
     """
 
     def __init__(self, n_atoms=1, n_res=1, n_seg=1,
-                 attrs=None,
+                 attrs=[],
                  atom_resindex=None,
```

Comment:
https://www.codementor.io/python/tutorial/essential-python-interview-questions

questions 6. The others are also a good read for people new to python.